src/lib/jsonParseV2/parseBatchUpdate.cpp

```diff
@@ -113,7 +113,19 @@ std::string parseBatchUpdate(ConnectionInfo* ciP, BatchUpdate* burP)
     }
     else if (name == "actionType")
     {
-      burP->updateActionType.set(iter->value.GetString());
+      UpdateActionType uat;
+      uat.set(iter->value.GetString());
+
```

Comment:
Good point. Fixed in 32bf17d (without modiying set()... I prefer not to change a method that others points of the code may be using).